slurm/src/slurmcc/cli.py

```diff
@@ -129,6 +132,45 @@ def _slurm_node_name_completer(
             output_prefix = prefix
         return [output_prefix + x + "," for x in self.slurm_node_names]
 
+    def cost_parser(self, parser: ArgumentParser) -> None:
+        parser.add_argument("-s", "--start",  type=lambda s: datetime.strptime(s, '%Y-%m-%d'),
+                            default=date.today().isoformat(),
+                            help="Start time period (yyyy-mm-dd), defaults to current day.")
+        parser.add_argument("-e", "--end",  type=lambda s: datetime.strptime(s, '%Y-%m-%d'),
+                            default=date.today().isoformat(),
+                            help="End time period (yyyy-mm-dd), defaults to current day.")
+        parser.add_argument("-o", "--out", required=True, help="Fully qualified output filename")
+
+        #parser.add_argument("--partition", action='store_true', help="Show costs aggregated by partitions")
+        #parser.add_argument("--user", action='store_true', help="Show costs aggregated by user")
+        #parser.add_argument("-f", "--fmt", type=str, help="Comma separated list of formatting options")
+
+    def cost(self, config: Dict, start, end, out):
+        """
+        Cost analysis and reporting tool that maps Azure costs
+        to SLURM Job Accounting data. This is an experimental
+        feature.
+        """
+        config['cache_root'] = "/tmp"
```

Comment:
1) What if the cx wants to override this? 
2) If it can be changed, it should be under a different namespace - like config.get("cost", {}).get("cache_root", "/tmp")

Reply:
Changed, this now comes from initconfig. If not present only then it defaults to /tmp